### PR TITLE
Helm Chart configuration values

### PR DESCRIPTION
**Pull request by tr0njavolta**
Adds helm chart configuration values.

Still needs:

- [ ] Examples
- [x] Hierarchy check - does the way this is presented make sense?

### CODE REVIEW

File: content/spaces/helm-reference.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+[space](#space)
+
+[certificates](#certificates)
+
+[router](#router)
+
+[billing](#billing)
+
+[features](#features)
```
Comment (plumbis) [inline content/spaces/helm-reference.md:29]: I'd drop this and let the TOC do the work.

File: content/spaces/helm-reference.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+|-----|------|---------|-------------|
+| version | string | `"0.1.0"` | Overall artifact version that affects xpkgs and related components. |
+{{</ table >}}
+
+### ingress
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| ingress | object | | Configurations for external requests coming into the space. |
```
Comment (plumbis) [inline content/spaces/helm-reference.md:62]: I wouldn't include keys that don't have configurable settings. Leave it up to the child keys like `ingress.host`

File: content/spaces/helm-reference.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+Configuration values for the Space router deployment.
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| router.oidc | list | `[]` | Configures [OpenID Connect Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens). <br> Current supported arguments for this array are a subset of the arguments defined in the above document. Those arguments include:<br>`--oidc-issuer-url` <br> `--oidc-client-id` <br> `--oidc-username-claim` |
+{{</ table >}}
+
+### billing
```
Comment (plumbis) [inline content/spaces/helm-reference.md:109]: I'd suggest sorting the sections alphabetically

File: content/spaces/helm-reference.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| version | string | `"0.1.0"` | Overall artifact version that affects xpkgs and related components. |
+{{</ table >}}
+
+### ingress
+
+{{< table >}}
```
Comment (plumbis) [inline content/spaces/helm-reference.md:59]:
```suggestion
{{< table table-sm table-striped>}}
```

Consider using `table-sm` and/or `table-striped` for formatting options. No opinions from me but wanted to make you aware of the extra table options.

**Review by plumbis — COMMENTED** [review]
I think the display format is a-okay. I'm not sure how else we could present.

**Review by turkenh — APPROVED** [review]
I feel like we should eventually document all the available parameters and even better automate that with a tool like https://github.com/norwoodj/helm-docs. 

However, this is already a great start, thanks @tr0njavolta 🙌

**Pull request merged**